openmc/data/data.py

```diff
@@ -0,0 +1,385 @@
+    'Ir193': 0.627, 'Pt190': 0.00012, 'Pt192': 0.00782,
+    'Pt194': 0.32864, 'Pt195': 0.33775, 'Pt196': 0.25211,
+    'Pt198': 0.07356, 'Au197': 1.0, 'Hg196': 0.0015,
+    'Hg198': 0.1004, 'Hg199': 0.1694, 'Hg200': 0.2314,
+    'Hg201': 0.1317, 'Hg202': 0.2974, 'Hg204': 0.0682,
+    'Tl203': 0.29524, 'Tl205': 0.70476, 'Pb204': 0.014,
+    'Pb206': 0.241, 'Pb207': 0.221, 'Pb208': 0.524,
+    'Bi209': 1.0, 'Th230': 0.0002, 'Th232': 0.9998,
+    'Pa231': 1.0, 'U234': 0.000054, 'U235': 0.007204,
+    'U238': 0.992742
+}
+
+ATOMIC_SYMBOL = {0: 'n', 1: 'H', 2: 'He', 3: 'Li', 4: 'Be', 5: 'B', 6: 'C',
+                 7: 'N', 8: 'O', 9: 'F', 10: 'Ne', 11: 'Na', 12: 'Mg', 13: 'Al',
+                 14: 'Si', 15: 'P', 16: 'S', 17: 'Cl', 18: 'Ar', 19: 'K',
+                 20: 'Ca', 21: 'Sc', 22: 'Ti', 23: 'V', 24: 'Cr', 25: 'Mn',
+                 26: 'Fe', 27: 'Co', 28: 'Ni', 29: 'Cu', 30: 'Zn', 31: 'Ga',
+                 32: 'Ge', 33: 'As', 34: 'Se', 35: 'Br', 36: 'Kr', 37: 'Rb',
+                 38: 'Sr', 39: 'Y', 40: 'Zr', 41: 'Nb', 42: 'Mo', 43: 'Tc',
+                 44: 'Ru', 45: 'Rh', 46: 'Pd', 47: 'Ag', 48: 'Cd', 49: 'In',
+                 50: 'Sn', 51: 'Sb', 52: 'Te', 53: 'I', 54: 'Xe', 55: 'Cs',
+                 56: 'Ba', 57: 'La', 58: 'Ce', 59: 'Pr', 60: 'Nd', 61: 'Pm',
+                 62: 'Sm', 63: 'Eu', 64: 'Gd', 65: 'Tb', 66: 'Dy', 67: 'Ho',
+                 68: 'Er', 69: 'Tm', 70: 'Yb', 71: 'Lu', 72: 'Hf', 73: 'Ta',
+                 74: 'W', 75: 'Re', 76: 'Os', 77: 'Ir', 78: 'Pt', 79: 'Au',
+                 80: 'Hg', 81: 'Tl', 82: 'Pb', 83: 'Bi', 84: 'Po', 85: 'At',
+                 86: 'Rn', 87: 'Fr', 88: 'Ra', 89: 'Ac', 90: 'Th', 91: 'Pa',
+                 92: 'U', 93: 'Np', 94: 'Pu', 95: 'Am', 96: 'Cm', 97: 'Bk',
+                 98: 'Cf', 99: 'Es', 100: 'Fm', 101: 'Md', 102: 'No',
+                 103: 'Lr', 104: 'Rf', 105: 'Db', 106: 'Sg', 107: 'Bh',
+                 108: 'Hs', 109: 'Mt', 110: 'Ds', 111: 'Rg', 112: 'Cn',
+                 113: 'Nh', 114: 'Fl', 115: 'Mc', 116: 'Lv', 117: 'Ts',
+                 118: 'Og'}
+ATOMIC_NUMBER = {value: key for key, value in ATOMIC_SYMBOL.items()}
+
+_ATOMIC_MASS = {}
+
+_GND_NAME_RE = re.compile(r'([A-Zn][a-z]*)(\d+)((?:_[em]\d+)?)')
+
+
+def atomic_mass(isotope):
+    """Return atomic mass of isotope in atomic mass units.
+
+    Atomic mass data comes from the `Atomic Mass Evaluation 2016
+    <https://www-nds.iaea.org/amdc/ame2016/AME2016-a.pdf>`_.
+
+    Parameters
+    ----------
+    isotope : str
+        Name of isotope, e.g., 'Pu239'
+
+    Returns
+    -------
+    float
+        Atomic mass of isotope in [amu]
+
+    """
+    if not _ATOMIC_MASS:
+
+        # Load data from AME2016 file
+        mass_file = os.path.join(os.path.dirname(__file__), 'mass16.txt')
+        with open(mass_file, 'r') as ame:
+            # Read lines in file starting at line 40
+            for line in itertools.islice(ame, 39, None):
```

Comment:
Is it worth pre-processing this file and creating something like  a `.csv` file that could be more easily read?